PICMI_Python/particles.py

```diff
@@ -282,6 +282,77 @@ def __init__(self, density,
         self.handle_init(kw)
 
 
+class PICMI_FoilDistribution(_ClassWithInit):
+    """
+    describes a foil with optional exponential pre-plasma ramps (front lower y/back higher y)
+
```

Comment:
This text is needed for the documentation to be properly formatted in html.
```suggestion
    Parameters
    ----------
```